src/testUtils/TestRig.react.js

```diff
@@ -57,10 +57,17 @@ export default class TestRig {
     var element = this.domNode.find(selector)
     this.fillInElement(element, value)
   }
-  toggleCheckbox(selector) {
-    var element = this.domNode.find(selector) // Selector should be specific to the checkbox
-    $(element).prop('checked', !($(element)[0].checked))
-    TestUtils.Simulate.change(element[0])
+  toggleCheckbox(element) {
+    element.checked = !(element.checked)
+    TestUtils.Simulate.change(element)
+  }
+  toggleRadioButton(element) {
+    element.checked = !(element.checked)
+    TestUtils.Simulate.change(element)
+  }
+  select(element, value) {
```

Comment:
@andela-tkomolafe Now that I see this method, though, you can go even farther: this method is actually just setting the value of an element--not just a `select`. So, this can be renamed to `setValue` and accept a `selector` instead of an `element`. Then `toggleElement` can be based on this method: 
```jsx
this.toggleElement(selector) {
  this.setValue(selector, !this.domNode.find(selector)[0].checked)
}
```